packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -18,16 +13,17 @@ import {
   LanguageClientOptions,
   RevealOutputChannelOn
 } from 'vscode-languageclient';
-import { LSP_ERR, VSCODE_APEX_EXTENSION_NAME } from './constants';
+import { LSP_ERR } from './constants';
 import { soqlMiddleware } from './embeddedSoql';
-import { languageServerUtils } from './helpers/languageServerUtils';
 import { nls } from './messages';
 import * as requirements from './requirements';
 import { telemetryService } from './telemetry';
 
 const UBER_JAR_NAME = 'apex-jorje-lsp.jar';
 const JDWP_DEBUG_PORT = 2739;
 const APEX_LANGUAGE_SERVER_MAIN = 'apex.jorje.lsp.ApexLanguageServerLauncher';
+const SUSPEND_LANGUAGE_SERVER_STARTUP =
+  process.env.SUSPEND_LANGUAGE_SERVER_STARTUP === 'true';
```

Comment:
i don't think everyone else wants this enabled by default (you and I do 😂 )

Reply:
@lukeis this is not setting the default, just a boolean check to set the correct value.